agent/util-scripts/gold/test-client-tool-meister/test-53.txt

```diff
@@ -201,21 +199,27 @@ testhost.example.com 0000 INFO pbench-tool-meister sysinfo -- pbench-sysinfo-dum
-testhost.example.com 0004 INFO pbench-tool-meister stop -- mpstat: stop_tool -- /var/tmp/pbench-test-utils/opt/pbench-agent/tool-scripts/mpstat --stop --dir=/var/tmp/pbench-test-utils/pbench/mock-run/0-iter-zero/sample42/tools-default/testhost.example.com --interval=42 --options=forty-two
-testhost.example.com 0005 INFO pbench-tool-meister wait -- Waiting for transient tool mpstat stop process
-testhost.example.com 0006 INFO pbench-tool-meister wait -- Waiting for transient tool mpstat start process
-testhost.example.com 0007 INFO pbench-tool-meister send_tools -- testhost.example.com: send_tools (no-op) default /var/tmp/pbench-test-utils/pbench/mock-run/0-iter-zero/sample42/tools-default/testhost.example.com
-testhost.example.com 0008 INFO pbench-tool-meister start -- mpstat: start_tool -- /var/tmp/pbench-test-utils/opt/pbench-agent/tool-scripts/mpstat --start --dir=/var/tmp/pbench-test-utils/pbench/mock-run/1-iter-one/sample42/tools-default/testhost.example.com --interval=42 --options=forty-two
-testhost.example.com 0009 INFO pbench-tool-meister stop -- mpstat: stop_tool -- /var/tmp/pbench-test-utils/opt/pbench-agent/tool-scripts/mpstat --stop --dir=/var/tmp/pbench-test-utils/pbench/mock-run/1-iter-one/sample42/tools-default/testhost.example.com --interval=42 --options=forty-two
-testhost.example.com 0010 INFO pbench-tool-meister wait -- Waiting for transient tool mpstat stop process
-testhost.example.com 0011 INFO pbench-tool-meister wait -- Waiting for transient tool mpstat start process
-testhost.example.com 0012 INFO pbench-tool-meister send_tools -- testhost.example.com: send_tools (no-op) default /var/tmp/pbench-test-utils/pbench/mock-run/1-iter-one/sample42/tools-default/testhost.example.com
-testhost.example.com 0013 INFO pbench-tool-meister stop -- Terminate issued for persistent tool dcgm
-testhost.example.com 0014 INFO pbench-tool-meister wait -- Stopped persistent tool dcgm
-testhost.example.com 0015 WARNING pbench-tool-meister end_tools -- testhost.example.com: unexpected temp files dcgm/dcgm-exporter.file
-testhost.example.com 0016 INFO pbench-tool-meister sysinfo -- pbench-sysinfo-dump -- /var/tmp/pbench-test-utils/opt/pbench-agent/unittest-scripts/pbench-sysinfo-dump /var/tmp/pbench-test-utils/pbench/mock-run/sysinfo/end/testhost.example.com block,security_mitigations,sos parallel
-testhost.example.com 0017 INFO pbench-tool-meister sysinfo -- testhost.example.com: sysinfo send (no-op) default /var/tmp/pbench-test-utils/pbench/mock-run/sysinfo/end/testhost.example.com
-testhost.example.com 0018 INFO pbench-tool-meister __exit__ -- testhost.example.com: terminating
+testhost.example.com 0004 INFO pbench-tool-meister.logger-start log_subprocess_output -- mpstat: running "/var/tmp/pbench-test-utils/opt/pbench-agent/tool-scripts/datalog/mpstat-datalog 42 --options=forty-two "
```

Comment:
We should consider masking the "timestamps" (like `0004`, here):  they make the differences much harder to review, because inserting or removing a single line in the sequence causes all subsequent "timestamps" to be different, which makes otherwise unchanged lines appear to be different, and that makes it hard to see which lines were actually changed.

Reply:
Those are not time stamps. those are the sequence order of logs.  We could consider dropping that for unit tests, but this highlights that out-of-order execution can and will occur (see failed test in PR #2697).

Reply:
I could have been confused at the time, but I'm pretty sure that the code that generates those thinks it's producing timestamps (I believe it's a mock), and it goes to some trouble to make sure that they are all different and monotonically increasing....

Reply:
See https://github.com/distributed-system-analysis/pbench/blob/main/lib/pbench/agent/redis.py#L212.

Reply:
Awesome!  You removed it!  Thanks!!

Reply:
Of course...since the numbers are still there in the gold file, it doesn't help so much on _this_ review...but the _next_ one will be easier.  😝

Reply:
> See https://github.com/distributed-system-analysis/pbench/blob/main/lib/pbench/agent/redis.py#L212.

Thanks for the reference!  That wasn't the one that I was thinking of (but, I cannot find the one that I was thinking of, now, so perhaps it was all a dream).